src/main/java/nl/rijksoverheid/mev/gezagsmodule/domain/gezagvraag/AdoptiefOuders.java

```diff
@@ -3,32 +3,35 @@
 import nl.rijksoverheid.mev.gezagsmodule.domain.Persoonslijst;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
 
 /**
  * v2a_2
  * "Ja" als is adoptief ouders, anders "Nee"
  */
-public class AdoptiefOuders extends GezagVraag {
+@Component
+public class AdoptiefOuders implements GezagVraag {
 
     private static final Logger logger = LoggerFactory.getLogger(AdoptiefOuders.class);
-
+    private static final String QUESTION_ID = "v2a.2";
     private static final String V2A_2_JA = "Ja";
     private static final String V2A_2_NEE = "Nee";
 
-    protected AdoptiefOuders(final GezagsBepaling gezagsBepaling) {
-        super(gezagsBepaling);
-        currentQuestion = "v2a.2";
+    @Override
+    public String getQuestionId() {
+        return QUESTION_ID;
```

Comment:
De constante wordt maar op één plek gebruikt. Wat mij betreft hoeft het dan geen constante te zijn.

```suggestion
        return "v2a.2";
```

Dit vind ik makkelijker lezen, omdat je de waarde van de constante niet op hoeft te zoeken. De naam van de getter geeft al aan dat het om een `question id` gaat. Wanneer het uit de hand loopt (meer dan 3 duplicate literals), herinnert Sonar je er wel aan.

Reply:
wordt op meerdere plaatsen gebruikt